precommit/src/main/shell/docker-cleanup.sh

```diff
@@ -204,3 +204,8 @@ docker_cleanup
 RESULT=$?
 
 cleanup_and_exit ${RESULT}
+
+function trigger_errors
+{
+  echo ${unprotected} 
```

Comment:
shellcheck:8: note: Double quote to prevent globbing and word splitting. [SC2086]
shellcheck:8: warning: unprotected is referenced but not assigned. [SC2154]
whitespace::end of line